windows-gaming-driver/src/driver/control/codec.rs

```diff
@@ -50,3 +52,40 @@ impl Encoder for Codec {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use bytes::BytesMut;
+    use tokio_io::codec::Decoder;
+
+    macro_rules! test {
+        (fn $name:ident, with $data:expr, expect $expected:expr, len $len:expr) => (
```

Comment:
On top of that, note how the vast majority of macros in Rust are designed to look and feel like functions (`println`, `assert`, pretty much all of the std macros). There definitely appears to be a convention against this kind of clutter.